bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
```

Comment:
well.. I suppose @guenter's intention for `set -o nounset` was to make sure script doesn't continue with any unset variables and as a general safeguard.
as often the case, there are many way of doing the same thing.. I'm leaving the original `set` as second line of the script and updating the first `if` statement to use string substitution, which is compatible with `-o nounset` and will not result in _unbound variable_ error
```
if [ "${1:-absent}" == "absent" ] || [ "${1:0:1}" == '-' ]; then
```